### PR TITLE
Add in-memory mode in the cmd wrapper

### PR DESCRIPTION
**Pull request by xiao**
Hello @brentburg, @nicks, 

Please review the following commits I made in branch 'xiao-in-memory-cmd'.

e7e6c29c6cd62f587934956cb381078b0d3abb8b (2015-06-08 14:45:31 -0700)
Add in-memory mode in the cmd wrapper

R=@brentburg
R=@nicks

### CODE REVIEW

File: bin/launch_local_dynamo.js [hunk 1]
```diff
@@ -8,20 +8,15 @@ var flags = require('flags')
...
 var localDynamo = require('../lib/launch.js')
 
-flags.defineString('database_dir', '', 'The locaction for databases files')
+flags.defineString('database_dir', '', 'The location for databases files. Run in memory if omitted.')
 flags.defineNumber('port', 4567, 'A port to run DynamoDB Local')
 flags.parse()
 
-if (!flags.get('database_dir')) {
-  console.error('Missing database directory')
+if (flags.get('database_dir') && !fs.existsSync(flags.get('database_dir'))) {
```
Comment (nicks) [inline bin/launch_local_dynamo.js]: existsSync is deprecated

https://nodejs.org/api/fs.html#fs_fs_existssync_path

Comment (nicks) [inline bin/launch_local_dynamo.js]: (fwiw, i think it's dumb that they deprecated it, but _shrug_)

**Comment by nicks** [comment]
lgtm

File: bin/launch_local_dynamo.js (hunk 1 above)
Comment (xiao) [inline bin/launch_local_dynamo.js]: Sad.

Updated. Removed the usage of `existSync`. Just yolo call `spawn()` and handle the `error` event instead.

**Comment by brentburg** [comment]
:+1:

**Pull request merged**